pf-azure/pf-resource-az/resources/pf-log-analytics/functions/public/manager/get-pfloganalytics.ps1

```diff
@@ -0,0 +1,28 @@
+Function Get-PfLogAnalytics
+{
+    Param (
+        [Parameter(Mandatory = $false)] 
+        [string] $Name,
+        [Parameter(Mandatory = $false)] 
+        [string] $SubscriptionName,
+        [Parameter(Mandatory = $false)] 
+        [string] $ResourceGroupName
+    )
+
```

Comment:
I think worth factoring this if null then getname into utility that takes in classprefix, string name, and maybe currentbuild (if not using global)